client/chain/chain.go

```diff
@@ -656,6 +660,19 @@ func (c *chainClient) runBatchBroadcast() {
 	}
 }
 
+func (c *chainClient) GetGasFee() (string, error) {
+	gasPrices := strings.Trim(c.opts.GasPrices, "inj")
+
+	gas, _ := strconv.ParseFloat(gasPrices, 18)
+
+	gasFeeAdjusted := gas * float64(c.gasWanted) / math.Pow(10, 18)
+	gasFeeFormatted := strconv.FormatFloat(gasFeeAdjusted, 'f', -1, 64)
+	c.gasFee = gasFeeFormatted
+	log.Debugln("gas fee: ", c.gasFee+" "+"INJ")
```

Comment:
Looks like this function is an utility, we don't need to log debug here:
- The c.gasFee will finally be returned, which can be logged outside the function
- Redundant log is annoying